### PR TITLE
@nivo/colors typescript migration

### PR DESCRIPTION
**Pull request by plouc**

### CODE REVIEW

**Review by wyze — CHANGES_REQUESTED** [review]
Looks fine, just had a question about peerDep range.

**Comment by wyze** [comment]
Oh also, did you want to add a few tests?

File: packages/colors/package.json [hunk 1]
```diff
@@ -8,23 +8,30 @@
...
+    "@types/d3-color": "^2.0.0",
+    "@types/d3-scale-chromatic": "^2.0.0",
+    "@types/lodash": "^4.14.149",
+    "@types/react-motion": "^0.0.29"
+  },
   "peerDependencies": {
+    "@nivo/core": "^0.64.0",
     "prop-types": ">= 15.5.10 < 16.0.0",
-    "react": ">= 16.8.4 < 17.0.0"
+    "react": ">= 16.8.4 <= 17.0.0"
```
[the remark this answers is not in the document]
Comment (plouc) [reply packages/colors/package.json:34]: Yea, we have an open issue about this, but I think we could tackle this as a separate PR, changing it for every package and maybe also adding @nivo/core to the devDependencies when relevant.

**Comment by plouc** [comment]
@wyze, I already created a complete test suite for this package: https://github.com/plouc/nivo/tree/master/packages/colors/tests

**Comment by wyze** [comment]
My fault!

**Review by wyze — APPROVED** [review]
Looks great!

**Pull request merged**